docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -33,28 +30,28 @@ $ ./benchmark_app –d CPU –m <model> -i <input>
+While these settings provide really low-level control and allow to leverage the optimal model performance on the _specific_ device, it is suggested to always start the performance evaluation with the [OpenVINO High-Level Performance Hints](../../OV_Runtime_UG/performance_hints.md) first:
  - benchmark_app **-hint tput** -d 'device' -m 'path to your model'
  - benchmark_app **-hint latency** -d 'device' -m 'path to your model'
 
 ## Comparing Performance with Native/Framework Code 
 
 When comparing the OpenVINO Runtime performance with the framework or another reference code, make sure that both versions are as similar as possible:
 
--	Wrap exactly the inference execution (refer to the  [Benchmark App](../../../samples/cpp/benchmark_app/README.md) for examples).
+-	Wrap exactly the inference execution (for examples, refer to the [Benchmark App](../../../samples/cpp/benchmark_app/README.md)).
 -	Do not include model loading time.
--	Ensure the inputs are identical for the OpenVINO Runtime and the framework. For example, beware of random values that can be used to populate the inputs.
--	Consider [Image Pre-processing and Conversion](../../OV_Runtime_UG/preprocessing_overview.md), while any user-side pre-processing should be tracked separately.
--   When applicable, leverage the [Dynamic Shapes support](../../OV_Runtime_UG/ov_dynamic_shapes.md)
--	If possible, demand the same accuracy. For example, TensorFlow allows `FP16` execution, so when comparing to that, make sure to test the OpenVINO Runtime with the `FP16` as well.
+-	Ensure the inputs are identical for the OpenVINO Runtime and the framework. For example, watch out for random values that can be used to populate the inputs.
+-	In situations when any user-side pre-processing should be tracked separately, consider [Image Pre-processing and Conversion](../../OV_Runtime_UG/preprocessing_overview.md).
+-  When suitable, leverage the [Dynamic Shapes support](../../OV_Runtime_UG/ov_dynamic_shapes.md)
+-	If possible, demand the same accuracy. For example, TensorFlow allows *`FP16`* execution, so when comparing to that, make sure to test the OpenVINO Runtime with the *`FP16`* as well.
 
 ## Internal Inference Performance Counters and Execution Graphs <a name="performance-counters"></a>
-Further, finer-grained insights into inference performance breakdown can be achieved with device-specific performance counters and/or execution graphs.
-Both [C++](../../../samples/cpp/benchmark_app/README.md) and [Python](../../../tools/benchmark_tool/README.md) versions of the `benchmark_app` supports a `-pc` command-line parameter that outputs internal execution breakdown.
+Deeper, more detailed insights into inference performance breakdown can be achieved with device-specific performance counters and/or execution graphs.
```

Comment:
```suggestion
More detailed insights into inference performance breakdown can be achieved with device-specific performance counters and/or execution graphs.
```
I'd say this much emphasis is enough for a word "insight", which already suggests discernment.